orca-webhook/src/main/java/com/netflix/spinnaker/orca/webhook/config/WebhookConfiguration.java

```diff
@@ -114,11 +114,18 @@ public ClientHttpRequestFactory webhookRequestFactory(
 
                   validateResponseSize(response, webhookProperties.getMaxResponseBytes());
 
-                  if (webhookProperties.isVerifyRedirects() && response.isRedirect()) {
-                    // verify that we are not redirecting to a restricted url
+                  if (response.isRedirect()) {
```

Comment:
So I see we have this `validateRequestSize` and `validateResponseSize`, would it make sense to move this to a method called `validateAnyRedirects`?

Reply:
We didn't decide the `isRedirect` method name here.  It's a method on the okhttp3.Response class.

We do get to decide property names though, and I figured `followRedirects` was consistent with the existing `verifyRedirects`.

Reply:
Aaah, I see...We could move all this logic to a separate method for handling redirect validation...yes, that makes sense.